event-checklist.md

```diff
@@ -0,0 +1,104 @@
+#Event committee checklist
+##Introduction
+Teams often run events under the auspices of [Linux Australia](https://www.linux.org.au). This document articulates the milestones you need to check off to ensure compliance with Linux Australia policies, good operations management - and to deliver a great event.
+
+##Overview
+The steps can be summarised as;
+1. Form an event subcommittee under the [event subcommittee policy](subcommittee_policy_v2.md).  
+2. Understand financial arrangements and set up banking access
+3. Put in place event infrastructure
+4. Put in place event Code of Conduct
+5. Ensure insurance coverage for event
+6. Ensure recognition of Linux Australia in event's digital presence
+7. Ensure Sponsorship Agreement is in place
+8. Ensure Post Event Report is written and distributed
+
+##Checklist for Event Teams
+* [ ] Get your event team together and ensure you have a balanced mix of skills
+* [ ] Read through the [event subcommittee policy](subcommittee_policy_v2.md)
+* [ ] Ensure all members of your team are [members of Linux Australia](https://www.linux.org.au/membership)
+* [ ] Raise a motion to form a new subcommittee by [emailing the Council](mailto:council@linux.org.au). A draft motion is provided below.

```

Comment:
I actually think the motion needs to be raised by the council. So this item should be "put forward a proposal".

We should also expand on what we expect in the proposal. In general we don't require anything flashy (certainly nothing like an LCA bid). It needs to state:
 - The goal of the subcommittee (eg to run $event in $location)
 - The proposed team members and their roles (according toe the subcommittee role types)
 - A rough outline of the event. (eg 3 day, 2 streams, 1 dinner, unconf etc)
 - A rough initial budget. The budget will go through the approval process of the subcommittee policy so there isn't a need for it to be rigorous at this stage. In general it should show viability of a conference with low sponsorship and conservative attendance.